randovania/data/json_data/prime3.txt

```diff
@@ -2147,12 +2534,17 @@ Asset id: 10980580650394598183
 > Dock to Imperial Crypt; Heals? False
   * Normal Door to Imperial Crypt/Dock to Imperial Caverns
   > Dock to Warp Site Bravo
-      Plasma Beam and Use Screw Attack (Space Jump)
+      After Imperial Caverns Ice Floors and Use Screw Attack (Space Jump)
 
 > Dock to Warp Site Bravo; Heals? False
   * Normal Door to Warp Site Bravo/Dock to Imperial Caverns
   > Dock to Imperial Crypt
-      Plasma Beam
+      After Imperial Caverns Ice Floors
+
+> Event - Ice Floors; Heals? False
+  * Event Imperial Caverns Ice Floors
+  > Dock to Warp Site Bravo
+      Trivial
 
 ====================
 Bryyo - Seed
```

Comment:
Add node called "Samus Ship", with "Heals?" checked, and have it connect to the door as "Trivial"